wcfsetup/install/files/js/3rdParty/pica.js

```diff
@@ -0,0 +1,2223 @@
+/* (The MIT License)
```

Comment:
pica is automatically pulled into `WoltLabSuite.Core.min.js` by r.js / when running _buildCore.js, because it is loaded as a require module. The same happens to favico, perfect-scrollbar and enquire.

Reply:
✔️